set.mm

```diff
@@ -61377,9 +61440,38 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
       ( cdm fdmd sseqtrid ) AEHDBFABCEGIJ $.
   $}
 
+  $( The image of a class is a subset of its codomain.  (Contributed by Glauco
+     Siliprandi, 17-Aug-2020.) $)
+  fimass $p |- ( F : A --> B -> ( F " X ) C_ B ) $=
+    ( wf cima crn imassrn frn sstrid ) ABCECDFCGBCDHABCIJ $.
+
+  $( The preimage of the codomain of a mapping is the mapping's domain.
+     (Contributed by FL, 25-Jan-2007.)  (Proof shortened by AV,
+     20-Sep-2024.) $)
```

Comment:
```suggestion
  $( The preimage of the codomain of a function with domain and codomain is the function's domain.
     (Contributed by FL, 25-Jan-2007.)  (Proof shortened by AV,
     20-Sep-2024.) $)
```

Reply:
I will replace "mapping" by "function" here, but this is a case in which "with domain and codomain" is not necessary, see my comment above.